### PR TITLE
disable progress bar in tests

### LINKED ISSUE #791: Disable progress bar in most tests
Output tracking does not gracefully deal with line rollback apparently, and adds another line upon each update. To avoid resulting overly long logfiles, it may be preferable to disable progress bars per default / in tests.

### PR DESCRIPTION
**Pull request by PaulJonasJost**
closes #791

### CODE REVIEW

File: test/optimize/test_optimize.py [hunk 1]
```diff
@@ -258,7 +258,7 @@ def get_optimizer(library, solver):
     elif library == 'fides':
         options[fides.Options.SUBSPACE_DIM] = solver[1]
         optimizer = optimize.FidesOptimizer(
-            options=options, hessian_update=solver[0], verbose=0
+            options=options, hessian_update=solver[0], verbose=40
```
Comment (yannikschaelte) [inline test/optimize/test_optimize.py:261]: I guess we'll want `verbose=logging.WARNING` or `logging.ERROR` or whatever 40 is.

Comment (PaulJonasJost) [reply test/optimize/test_optimize.py:261]: yes I think we changed it at the same time :D

File: pypesto/sample/parallel_tempering.py [hunk 2]
```diff
@@ -69,8 +70,10 @@ def initialize(
 
     def sample(self, n_samples: int, beta: float = 1.0):
         """Sample and swap in between samplers."""
+        show_progress = self.options['show_progress']
         # loop over iterations
-        for i_sample in tqdm(range(int(n_samples))):  # TODO test
+        for i_sample in tqdm(range(int(n_samples)), disable=not show_progress):
+            # TODO test
```
Comment (jvanhoefer) [inline pypesto/sample/parallel_tempering.py:76]: Is this still ToDo?

**Comment by jvanhoefer** [comment]
looks good!

File: pypesto/sample/parallel_tempering.py (hunk 2 above)
Comment (PaulJonasJost) [reply pypesto/sample/parallel_tempering.py:76]: not sure, was there before. Would probably have to ask @yannikschaelte

File: test/optimize/test_optimize.py (hunk 1 above)
Comment (yannikschaelte) [reply test/optimize/test_optimize.py:261]: ah, I meant literally writing the logging.XYZ, which is just an int of value 40, but improves readability.

File: pypesto/sample/parallel_tempering.py (hunk 2 above)
Comment (yannikschaelte) [reply pypesto/sample/parallel_tempering.py:76]: yeah I guess we wanted to add tests there, tbd independent of current pr

**Pull request merged**